<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NSRAboutDialog</class>
 <widget class="QWidget" name="NSRAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="NSRScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>388</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="versionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;	
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>NSR Reader 2.0.0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="developerLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Alexander Saprykin, (c) 2011-2015</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="contactsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;	
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Contacts: &lt;nsr.reader@gmail.com&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="iconsCreditLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;	
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>Used icons: Stuttgart pack, Axialis Team</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="licenseLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>48</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
         </property>
         <property name="text">
          <string>GPLv2 license</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgbs(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
	font: bold 16px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/icons/icons/twitter.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>www.twitter.com/NSRReader</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgbs(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
	font: bold 16px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resources.qrc">:/icons/icons/facebook.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="facebookLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>48</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	background-color: white;
	border: 1px solid rgba(180, 180, 180, 255);
	border-right:none;
	border-left:none;
	border-top:none;
	color: rgb(0, 0, 0);
}</string>
           </property>
           <property name="text">
            <string>Find on Facebook!</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NSRScrollArea</class>
   <extends>QScrollArea</extends>
   <header>nsrscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
